sql/moz-fx-data-marketing-prod/ga_derived/www_site_metrics_summary_v2/query.sql

```diff
@@ -0,0 +1,71 @@
+-- Query for ga_derived.www_site_metrics_summary_v2
+            -- For more information on writing queries see:
+            -- https://docs.telemetry.mozilla.org/cookbooks/bigquery/querying.html
+WITH site_data AS (
+  SELECT
+    PARSE_DATE('%Y%m%d', event_date) AS date,
+    device.category AS device_category,
+    device.operating_system AS operating_system,
+    device.web_info.browser AS browser,
+    device.language AS language,
+    geo.country AS country,
+    traffic_source.source AS source,
+    traffic_source.medium AS medium,
+    collected_traffic_source.manual_campaign_name AS campaign,
+    collected_traffic_source.manual_content AS ad_content,
+    SUM(CASE WHEN event_name = 'session_start' THEN 1 ELSE 0 END) AS sessions,
+    SUM(
+      CASE
+        WHEN event_name = 'session_start'
+          AND NOT `moz-fx-data-shared-prod.udf.ga_is_mozilla_browser`(device.web_info.browser)
+          THEN 1
+        ELSE 0
+      END
+    ) AS non_fx_sessions,
+    SUM(CASE WHEN event_name = 'product_download' THEN 1 ELSE 0 END) AS downloads,
+    SUM(
+      CASE
+        WHEN event_name = 'product_download'
+          AND NOT `moz-fx-data-shared-prod.udf.ga_is_mozilla_browser`(device.web_info.browser)
+          THEN 1
+        ELSE 0
+      END
+    ) AS non_fx_downloads
+  FROM
+    `moz-fx-data-marketing-prod.analytics_313696158.events_*`
+  WHERE
+    event_date = FORMAT_DATE('%Y%m%d', @submission_date)
```

Comment:
GA4 data has up to 3 days of late-arriving data: https://support.google.com/analytics/answer/7029846?#tables

I accounted for this in GA Sessions by having separate airflow tasks that first backfilled the last two days of data: https://github.com/mozilla/telemetry-airflow/blob/main/dags/ga_session_backfill.py

Reply:
Will make a similar new DAG for GA4 - thanks for the tip!